WordPress/WordPressShareExtension/MainShareViewController.swift

```diff
@@ -96,13 +96,9 @@ private extension MainShareViewController {
 
         let shareNavController = UINavigationController(rootViewController: editorController)
 
-        if #available(iOS 13, *), editorController.originatingExtension == .saveToDraft {
-            // iOS 13 has proper animations and presentations for share and action sheets. So the `else` case should be removed when iOS 13 is minimum.
-            // We  need to make sure we don't end up with stacked modal view controllers by using this:
+        if editorController.originatingExtension == .saveToDraft {
+            // We need to make sure we don't end up with stacked modal view controllers by using this:
             shareNavController.modalPresentationStyle = .overFullScreen
-        } else {
-            shareNavController.transitioningDelegate = extensionTransitioningManager
-            shareNavController.modalPresentationStyle = .custom
```

Comment:
@mokagio  you still need to keep this `else` clause and code path for the case when the source is not the SaveToDraft extension

Reply:
Oh wait, and now I just read the `// code comment` that was there before your change, and that said:
```
// iOS 13 has proper animations and presentations for share and action sheets. So the `else` case should be removed when iOS 13 is minimum.
```

So maybe you were right to remove the `else` clause after all! 🤔 (so sorry for the back-and-forth 😅  I should be more mindful of code comments in diffs)

Reply:
The release notes in that PR list

> * Fixed bugs with the "Save as Draft" action extension's navigation bar colors and iPad sizing in iOS 13.

Skimming the PR description, I think the `editorController.originatingExtension == .saveToDraft` check is intentional and required.

I'll verify the behavior for the two extensions and in both versions of the code and report. _Stay tuned._

Reply:
With the code at [`cd07203` (#19513)](https://github.com/wordpress-mobile/WordPress-iOS/pull/19513/commits/cd0720341688986aaabba72be007583cb607d9eb), I'm getting the following

**Save as Draft**

<table>
<tr>
	<td>

![IMG_4733](https://user-images.githubusercontent.com/1218433/205038233-73872a9d-0130-4749-9342-c061d09e050c.PNG)

</td>
	<td>

![IMG_4734](https://user-images.githubusercontent.com/1218433/205038246-26d7b735-0942-40b7-b1b6-efb93d3b17d5.PNG)

</td>
</tr>
</table>

**Share**

<img alt="IMG_4736" src="https://user-images.githubusercontent.com/1218433/205038441-d31bfe38-1425-4a65-97bb-49778c0c84a8.PNG" width=300/>

Share doesn't look right. Might need more work. The first thing I'll try will be removing the `else` branch and add some break points. I'm also not sure why I'm getting that error, given the app was setup alright.

Reply:
> Share doesn't look right. Might need more work.

https://github.com/wordpress-mobile/WordPress-iOS/pull/19700